DuckDuckGo/Info.plist

```diff
@@ -205,6 +205,8 @@
 	<array>
 		<string>arm64</string>
 	</array>
+	<key>UIStatusBarHidden</key>
```

Comment:
This is mainly here to hide on the launch screen - seems to be OK in the app as we override the status bar preferences in the view controllers for the theme.